buildkite/resource_pipeline.go

```diff
@@ -295,9 +295,9 @@ func preparePipelineRequestPayload(d *schema.ResourceData) *Pipeline {
 	for k, vI := range d.Get("env").(map[string]interface{}) {
 		req.Environment[k] = vI.(string)
 	}
-	req.ProviderSettings = map[string]bool{}
+	req.ProviderSettings = map[string]string{}
```

Comment:
Sorry, I am not following - you mean the `.tfstate` file? Also, would you mind dropping me an email at my firstname at my lastname dot com, please? :)